test/prism/fixtures/tilde_heredocs.txt

```diff
@@ -95,3 +95,13 @@ EOF
   #{1}
 	b
 EOT
+
+<<~'  FOO'
```

Comment:
The tests are failing because CRuby couldn't handle this case until Ruby 3.3. So it's going to fail for all cases < 3.3. In this case could you pull these examples into a different file (including the dash heredocs) and then explicitly ignore that fixture if < 3.3? We do something similar in `parse_test.rb`.